site/content/contribute/webapp/end-to-end-tests.md

```diff
@@ -25,6 +25,27 @@ Not familiar with Cypress? Here is some documentation that will help you get sta
     - Initiate via node CLI to selectively run specs based on test metadata, e.g. `node run_tests.js --group='@accessibility'` which will run all specs with `@accessibility` metadata.
 3.  Tests are executed according to your selection and will display whether the tests passed or failed.
 
+### Environment Variables
+There are several environment variables we're using for Cypress testing with the primary purpose:
+- to easily change when running in CI, and
+- to cater different values across developer machines.
+
+Environment variables are defined in "[cypress.json](https://github.com/mattermost/mattermost-webapp/blob/master/e2e/cypress.json)" under `env` key. In most cases, you don't need to change the values as it make use of the default local developer setup, but when you do, the easiest is to override by exporting `CYPRESS_*`, where `*` is the key of the variable like `CYPRESS_adminUsername`. See [Cypress documentation](https://docs.cypress.io/guides/guides/environment-variables.html#Setting) for details.
+
+| Variable            | Description                                |
+|---------------------|--------------------------------------------|
+| CYPRESS\_adminUsername | Admin's username the test server.<br><br>Default: `"sysadmin"` when server is seeded by `make test-data`. |
```

Comment:
```suggestion
| CYPRESS\_adminUsername | Admin's username for the test server.<br><br>Default: *sysadmin* when server is seeded by `make test-data`. |
```

Reply:
Can we remain the sysadmin enclosed with backtick and double quote? With the suggested format, the default value is not being highlighted. See screenshot.
<img width="822" alt="Screen Shot 2020-10-05 at 8 22 09 PM" src="https://user-images.githubusercontent.com/5334504/95079225-e4a52500-0748-11eb-80f3-7b639e44371e.png">

Reply:
Absolutely! :)